flink-python/pyflink/table/table_environment.py

```diff
@@ -183,11 +183,29 @@ def insert_into(self, table, table_path, *table_path_continued):
         j_table_path = utils.to_jarray(gateway.jvm.String, table_path_continued)
         self._j_tenv.insertInto(table._j_table, table_path, j_table_path)
 
+    def list_catalogs(self):
+        """
+        Gets the names of all catalogs registered in this environment.
+
+        :return: List of catalog names.
+        """
+        j_catalog_name_array = self._j_tenv.listCatalogs()
+        return [item for item in j_catalog_name_array]
+
+    def list_databases(self):
+            """
```

Comment:
I'm wondering why the indention is different from others here. Not sure if it's a github showing rendering issue.

Reply:
this time it's actually a real indentation issue. I've updated https://github.com/apache/flink/pull/8829 and rebased this PR to #8829